documentation/code-reviews.md

```diff
@@ -0,0 +1,67 @@
+
+If there are particular circumstances, original ways of tackling something, or things you have deliberately left out of scope, mention those as well. Instructions like steps to reproduce a bug or specific test data may also save the reviewer time.
+
+### Add reviewers
+
+Make sure that the right people are designated as reviewers, or better yet, set Azure to do this automatically for you. We want each PR to be approved by at least one of the maintainers.
+
+### Add initial comments
+
+As an author, you can start a discussion about a piece of code by commenting on it yourself. For example, you can indicate that you’d like feedback about a specific code fragment.
+
+### Check the tests
+
+When saving the PR, GitHub will statically analyse the code for linting errors or failing tests. Check back after a few minutes to verify there have been no errors – otherwise, the PR cannot merge, which will block the first reviewer. Alternatively, run the tests locally: `npm run lint && npm run test`.
+
+### Consider the PR an intermediate step
+
+Creating a PR does not imply “My work is done. Would you please look at it?”. Consider it more like: “This is my proposal for implementing the task. Let’s use our shared knowledge and complete the work together.”
+
+If necessary, you can prepare the PR earlier; for example, to see a clear diff, have someone else watch or post comments or questions in advance. Then, put the PR in ‘draft’ mode so that no one receives notifications or thinks it is already time for the review.
+
+## Provide feedback on code
+
+Try to follow up on a pull request as quickly as possible. After all, we want to avoid long-living branches. The work will also be ‘top of mind’ for the author at the early stages. Make sure you receive notifications of new PRs in your name.
+
+Invest the time to understand the PR change fully. Read the title and description carefully. Typically, you will check out the branch locally and test the functionality explicitly. Make yourself familiar with the new code as if you had written it yourself.
+
+Use GitHub’s ‘Start a review’ feature to ensure the author receives only one notification for all your comments.
+
+### Add comments
+
+Please realise that your written feedback may come across as more critical than you intended.
+
+Your colleague has worked in good faith; make sure you communicate openly. Ask questions and make suggestions. “Would it be okay to…?” “Have you taken into account that…?” “I would expect such and such.”
+
+Call each other if that is more convenient. And write a compliment if you come across some quality work.
+
+### Suggest changes
+
+Use GitHub’s suggestion feature to propose specific changes to code. Though you’ll still want to introduce the change, suggestions are often more straightforward than more prose. The author can apply or reject the proposal at the touch of a button.
+
+### Resolve comments
+
+The author of the pull request is the one to resolve all comments.
```

Comment:
This is different from what we currently do, right? The reviewer is the one who resolves now. 

I prefer our current approach, it means the reviewer can do a final check of the implemented changes, and all merged code will be seen at least once by the reviewer

Reply:
Oh wow, we did agree to that approach and I did mean to express just that, but apparently wrote it just the other way round.